staker-rewards/lib/rewards-calculator.js

```diff
@@ -129,7 +129,10 @@ export default class RewardsCalculator {
       violations.push("poolAuthorizedAtStart")
     }
 
-    if (requirements.poolDeauthorizedInInterval === true) {
+    if (
+      requirements.poolDeauthorizedInInterval === true &&
+      operatorAssets.ethBonded.isEqualTo(new BigNumber(0))
+    ) {
```

Comment:
You're right but I think this way we communicate explicitly why there is no reward. If we detected deauthorization and no ETH is bonded, it will be clearly visible in the report.